OPHD/UI/StringTable.h

```diff
@@ -68,7 +70,7 @@ class StringTable
 	std::vector<CellWithPosition> mCells;
 	const std::size_t mColumnCount;
 	const std::size_t mRowCount;
-	NAS2D::Point<int> mPosition;
+	NAS2D::Rectangle<int> mScreenRect = { };
```

Comment:
`Rectangle` structs default to zero initialization. The explicit initialization here is not needed.